python/paddle/fluid/tests/unittests/test_complex_reshape.py

```diff
@@ -13,38 +13,80 @@
 # limitations under the License.
 
 import paddle.fluid as fluid
+import paddle
 from paddle import complex as cpx
 import paddle.fluid.dygraph as dg
 import numpy as np
 import unittest
 
 
 class TestComplexReshape(unittest.TestCase):
+    def setUp(self):
+        self._dtypes = ["float32", "float64"]
+        self._places = [paddle.CPUPlace()]
+        if fluid.core.is_compiled_with_cuda():
+            self._places.append(paddle.CUDAPlace(0))
+
     def test_case1(self):
-        x_np = np.random.randn(2, 3, 4) + 1j * np.random.randn(2, 3, 4)
-        shape = (2, -1)
+        for dtype in self._dtypes:
+            x_np = np.random.randn(
+                2, 3, 4).astype(dtype) + 1j * np.random.randn(2, 3,
+                                                              4).astype(dtype)
+            shape = (2, -1)
+            for place in self._places:
+                with dg.guard(place):
+                    x_var = dg.to_variable(x_np)
+                    y_var = cpx.reshape(x_var, shape)
+                    y_np = y_var.numpy()
+                    np.testing.assert_allclose(np.reshape(x_np, shape), y_np)
+
+    def test_case2(self):
```

Comment:
we don't recommend naming by `1,2,3,4...`, the original code style is not good, maybe you can polish test_case's name in next pr

Reply:
ok, I will change the test_case's name in next pr.